lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -6,35 +6,42 @@ module FirebaseAppDistributionAuthClient
       TOKEN_CREDENTIAL_URI = "https://oauth2.googleapis.com/token"
 
       def fetch_auth_token(google_service_path)
-        if ENV["XDG_CONFIG_HOME"]
-          config_path = File.expand_path("configstore/firebase-tools.json", ENV["XDG_CONFIG_HOME"])
-        else
-          config_path = File.expand_path(".config/configstore/firebase-tools.json", "~")
-        end
+        config_path = find_firebase_tools
```

Comment:
Since this variable is only used in one place, you can remove this and just call the method in-line.

Reply:
Oops was looking at an old version